rest/model/project.go

```diff
@@ -106,6 +106,8 @@ type APIPatchTriggerDefinition struct {
 	// Name of the module corresponding to the upstream project in the
 	// downstream project's YAML.
 	ParentAsModule *string `json:"parent_as_module,omitempty"`
+	// The revision at which to create the downstream patch.
+	DownstreamRevision *string `json:"downstream_revision,omitempty"`
```

Comment:
It looks like this API model is also used by GQL resolvers - just to check, if a user does something like save the triggers in project settings, would it clear this field or preserve it? I just fixed an issue where GQL didn't propagate a field and it accidentally cleared the setting each time it saved the project settings section, so just wanted to double-check.

Reply:
That's a great catch -- thank you for flagging. I created a [Spruce PR](https://github.com/evergreen-ci/evergreen/pull/8012) as that's the only way to really get around this.